src/django/api/tests/test_location_contribution_strategy.py

```diff
@@ -70,7 +76,7 @@ def test_source_set_as_api_regardless_of_whether_passed(self, mock_get):
         self.assertNotIn('source', self.common_valid_input_data)
 
         event_dto = CreateModerationEventDTO(
-            contributor_id=self.contributor,
```

Comment:
Why did you decide to remove `_id` suffix for the contributor?

Reply:
Because it is a Contributor instance.